randovania/games/prime_hunters/layout/force_field_configuration.py

```diff
@@ -0,0 +1,178 @@
+from __future__ import annotations
+
+import copy
+import dataclasses
+from enum import StrEnum
+from typing import TYPE_CHECKING, Self
+
+from randovania.bitpacking import bitpacking
+from randovania.bitpacking.bitpacking import BitPackDecoder, BitPackEnum, BitPackValue
+from randovania.game.game_enum import RandovaniaGame
+from randovania.game_description.db.configurable_node import ConfigurableNode
+from randovania.game_description.db.node_identifier import NodeIdentifier
+from randovania.lib import enum_lib
+
+if TYPE_CHECKING:
+    from collections.abc import Iterator
+
+
+class LayoutForceFieldRequirement(BitPackEnum, StrEnum):
```

Comment:
make this just `Enum` unless you specifically need it to be `StrEnum` for something? (and if you do, pls lmk what)

Reply:
Uh no reason. It probably can just be an enum. Copy paste is fun. Dunno if it's cause issues below though.